public/static/docs/api-reference/open.md

```diff
@@ -0,0 +1,171 @@
+# dvc.api.open()
+
+[Context manager](https://www.python.org/dev/peps/pep-0343/#context-managers-in-the-standard-library)
+to open a tracked file.
+
+## Definition
+
+### Signature
+
+```py
+open(path, repo=None, rev=None, remote=None, mode="r", encoding=None)
+```
+
+### Types
+
+All **parameter** types are
+[string](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str).
+
+The **return** type is a
+[file object](https://docs.python.org/3/glossary.html#term-file-object).
+
+No exceptions are raised by this function directly.
```

Comment:
Why anybody would care about directly? The most common would be `PathMissingError` here.

Reply:
yep, agreed with @Suor ... you don't about "directly", to be precise you don't care at all how that function is implemented internally and where that exception comes from - some helper of the function itself.

Reply:
> Why anybody would care about directly?

The reason I used that word is not that I think people "care about" the internals. I'm trying to narrow down the explanation to just certain relevant exceptions. Just saying "No exceptions are raised by this function." will probably never be true for any function anywhere.

That said, I never liked the "directly" part anyway so I've removed it now. But I don't know how best to express the idea above then. Do you have a constructive suggestion @Suor? Perhaps a list of relevant exceptions we should mention here?

Reply:
Or if it's not an important point, then this is resolved.

Reply:
I think @Suor 's point was that this function actually raises some exceptions that we can mention, and it's not important if the come from the function or some internal helper

Reply:
K, I got the idea. I just don't know what are these exceptions. I should look into the code and play with the API to find them, but if you have the full list to confirm @Suor, that would be great.

Reply:
I added `PathMissingError` to the open() ref but looking at the core repo I can't detect any other relevant ones to add. There's a bunch actually, such as `dvc.config.ConfigError` which can happen when creating a `Repo` object, but seems sort of irrelevant to mention here? Then there's `CloneError`, `RevError`, `SCMError` (from `class Git`), as well as `NoRemoteInExternalRepoError`, `NoOutputInExternalRepoError`, and `NoOutputInExternalRepoError` (from `external_repo()`) but not sure whether they apply to API fns (how to produce them). 

It's a little consuming for me to do this code analysis AND determine which ones to add here given my current priorities (move faster on docs). I could def. use some help with a list of relevant exceptions users should be aware of for the API ref @Suor @shcheklein – anyone else I should mention here? Thanks 🙂